privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -489,7 +489,11 @@ def _ldap_attributes_to_user_object(self, attributes):
             for map_k, map_v in self.userinfo.items():
                 if ldap_k == map_v:
                     if ldap_k == "objectGUID":
-                        ret[map_k] = ldap_v[0]
+                        # An objectGUID should be no list, since it is unique
+                        if type(ldap_v) == basestring:
+                            ret[map_k] = ldap_v
```

Comment:
I think, for consistency, we should strip ``{`` and ``}`` here. Otherwise, we will get different values for ldap3 2.1.1 and ldap3 2.5.

Reply:
OK strip curly braces

Reply:
Done